operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2136,6 +2136,9 @@ nginx:
           location = /api/v1/push {
             proxy_pass      http://{{ template "mimir.fullname" . }}-distributor-headless.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.serverHttpListenPort" . }}$request_uri;
           }
+          location = /otlp {
```

The endpoint that mimir exposes is `/otlp/v1/metrics`
https://github.com/grafana/mimir/blob/3d14b39c7fde2f10a38875460aecd18ce639d0dd/pkg/api/api.go#L239

and the equals sign here signifies a strict match. We should match on a prefix instead
```suggestion
          location /otlp {
```

---

Added changes of helm tests